### PR TITLE
LPS-113302 Apply classic ckeditor implementation in Message Boards, part 1/2

### PR DESCRIPTION
**Pull request by markocikos**
Task history:
1. https://github.com/jbalsas/liferay-portal/pull/2221
2. https://github.com/liferay-frontend/liferay-portal/pull/107
3. https://github.com/liferay-frontend/liferay-portal/pull/121

Changes from https://github.com/liferay-frontend/liferay-portal/pull/121:
- Removed the ugly manual script loading. Instead, we are fixing the default `ckeditor4-react` loading, by deleting `CKEDITOR` property instead of assigning `undefined`. 
- Simplified `beforeScreenFlip` event handling, by using `Liferay.once`, instead of `on` and `detach`.

There is a `window.CKEDITOR = undefined` in [test of the CKEditor's jquery adapter](https://github.com/ckeditor/ckeditor4/blob/eb2d11644a796cd13cd047cd55541430359f9317/tests/adapters/jquery/manual/nockeditor.html), revealing that we are not supporting IE8 with this change. I sincerely hope that is not a problem :smile:

### CODE REVIEW

**Comment by markocikos** [comment]
ci:test:sf

**Comment by markocikos** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
I would ship this, LGTM!

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js [hunk 1]
```diff
@@ -87,18 +93,20 @@ const ClassicEditor = ({
...
 
-	useEventListener(
-		'resize',
-		() => setToolbarSet(getToolbarSet(initialToolbarSet)),
-		true,
-		window
-	);
+	const onResize = debounce(() => {
+		setToolbarSet(getToolbarSet(initialToolbarSet));
+	}, 200);
```
Comment (wincent) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js:98]: No need to send another rev to this, but for future reference, I usually name my magical constants to give them the illusion of non-arbitrariness.

```js
const SCIENTIFICALLY_PROVEN_OPTIMAL_DEBOUNCE_DELAY_IN_MS = 200;
```

😉

**Comment by markocikos** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
shipit

**Comment by jbalsas** [comment]
ci:forward

**Pull request closed without merging**